client/src/components/DataFiles/DataFilesModals/DataFilesRenameModal.js

```diff
@@ -55,24 +22,21 @@ const DataFilesRenameModal = () => {
     });
   };
 
-  const status = useSelector(state => state.files.operationStatus.rename);
-
   const history = useHistory();
   const location = useLocation();
   const reloadPage = (name, newPath) => {
     history.push(location.pathname);
-    updateSelected({ ...selectedFile, name, path: `/${newPath}` });
   };
 
-  const validate = e => {
-    setNewName(e.target.value);
-    const regexp = new RegExp(/["'/\\]/);
-    try {
-      setValidated(!regexp.test(e.target.value));
-    } catch {
-      setValidated(false);
-    }
-  };
+  const validationSchema = Yup.object().shape({
+    newName: Yup.string()
+      .min(1)
+      .matches(
+        /^[\d\w\-_.]+$/,
+        'Please enter a valid file name (accepted characters are A-Z a-z 0-9 - _ .)'
+      )
+      .required('Please enter a valid file name.')
+  });
```

Comment:
Should we add a validation check to make sure the string is different than the original filename?